modules/virt-creating-custom-monitoring-label-for-vms.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-creating-custom-monitoring-label-for-vms_{context}"]
+= Creating a custom monitoring label for virtual machines
+
+To enable queries to multiple virtual machines from a single service, create a custom label and assign it to the virtual machine's YAML file.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+* Edit the `template` spec of your virtual machine configuration file. The following example shows the label name `metrics` being added to the label `monitor`.
++
+[source,yaml]
+----
+spec:
+  template:
+    metadata:
+      labels:
+        monitor: metrics
+----
+
+.Verification
+
+You can verify that the `node-exporter` service is working. Metrics are exposed through an HTTP service endpoint under the `/metrics` canonical name.
+
+. Obtain the `Cluster-IP` for the `node-exporter` service.
++
+[source,terminal]
+----
+$ oc get service -n <node_exporter_service_name>
+----
++
+.Example output
+[source,terminal]
+----
+NAME                  TYPE      CLUSTER-IP     EXTERNAL-IP PORT(S)  AGE
+node-exporter-service ClusterIP 172.30.226.162 <none>      9100/TCP 4d22h
+
+----
+
+. To list all available metrics for the `node-exporter` service, use the `curl` query against `http://<endpoint>/metrics`.
++
+[source,terminal]
+----
+$ curl http://<172.30.226.9100>/metrics
+----
++
+.Example output
+[source,terminal]
+----
+# HELP app_connections number of connections for an app
```

Comment:
Same as my previous comment. These 4 app_connection lines wont be shown unless you create the custom monitor. They were defined by me. Not sure if you want to remove them or leave them in since it is just example output.

Reply:
Removed them since they're more like comments.